elements/bulbs-video/elements/meta/components/root.js

```diff
@@ -37,13 +37,24 @@ export default function VideoMetaRoot (props) {
     seriesElement = seriesNameElement;
   }
 
+  let campaign;
+  if (props.video.tunic_campaign_url) {
+    campaign = (
+      <VideoMetaCampaign
+          campaignPlacement={props.campaignPlacement}
+          campaignPreamble={props.campaignPreamble}
+          campaignTrackAction={props.campaignTrackAction}
+          campaignUrl={props.video.tunic_campaign_url} />
+    );
+  }
+
   return (
     <div className='bulbs-video-meta'>
       <div className='bulbs-video-meta-copy'>
 
         { seriesElement }
 
-        { props.campaignUrl ? <VideoMetaCampaign {...props}/> : null }
+        {campaign}
```

Comment:
@collin do you see any issues with using the sponsor from the video object itself instead of the url which is passed in from the site's CMS.

From what I've gathered AV is putting sponsorships in both places so these fields represent the same thing essentially. With onion studios though the video will show sponsorship across all usages instead of only on the site.